app/Http/Middleware/ShareInertiaData.php

```diff
@@ -27,6 +28,7 @@ public function handle($request, $next)
                 return [
                     'flash' => $request->session()->get('flash', []),
                     'languages' => LocaleHelper::getLocaleList(),
+                    'timezones' => TimezoneHelper::getListOfTimezones(),
```

Comment:
Are you sure? That's 400+ new entries every time a new HTTP request is made...

Reply:
Yes, actually I thought I removed it, it's not needed anymore.